oracle/config/oracle.go

```diff
@@ -13,22 +13,17 @@ import (
 // data providers for the currency pairs at the specified update interval.
 type OracleConfig struct {
 	// UpdateInterval is the interval at which the oracle will fetch prices from providers.
-	UpdateInterval time.Duration `mapstructure:"update_interval" toml:"update_interval"`
+	UpdateInterval time.Duration `json:"updateInterval"`
 
 	// Providers is the list of providers that the oracle will fetch prices from.
-	Providers []ProviderConfig `mapstructure:"providers" toml:"providers"`
-
-	// Market defines the market configurations for how currency pairs will be resolved to a
-	// final price. Each currency pair can have a list of convertable markets that will be used
-	// to convert the price of the currency pair to a common currency pair.
-	Market AggregateMarketConfig `mapstructure:"market" toml:"market"`
```

Comment:
How is the aggregation logic going to be configured now without AggregateMarketConfig?

Reply:
going to reference the marketmap module, which will have the relevant market information.